src/controllers/__tests__/cgnOperatorSearchController.test.ts

```diff
@@ -19,8 +19,14 @@ import {
   OrderingEnum
 } from "../../../generated/cgn-operator-search/OfflineMerchantSearchRequest";
 import { NonNegativeInteger } from "@pagopa/ts-commons/lib/numbers";
-import { CardActivated } from "../../../generated/io-cgn-api/CardActivated";
-import { CardExpired } from "../../../generated/io-cgn-api/CardExpired";
+import {
```

Comment:
These changes should not be in this PR.

Reply:
Moved to another PR (https://github.com/pagopa/io-backend/pull/918)